app/full_text_search.py

```diff
@@ -370,7 +370,8 @@ def create_document(person):
     # Applies two methods because kanji is used in Chinese and Japanese,
     # and romanizing in chinese and japanese is different.
     romanize_methods = [script_variant.romanize_word_by_unidecode,
-                        script_variant.romanize_japanese_word]
+                        script_variant.romanize_japanese_word,
+                        script_variant.romanize_chinese_name]
```

Comment:
Looks like it is expected that romanize_methods here returns a list. Can you update romanize_chinese_name to return a list (with zero or one string) instead of a string?